### ISSUE TITLE
Undefined array key "display_gpx" in version 14b

### ISSUE THREAD

**Issue by petitssuisses** [Piwigo/piwigo-openstreetmap#245]
Hi

I get this warning after last upgrade to version 14b
Warning: Undefined array key "display_gpx" in /var/www/xxx/photos/plugins/piwigo-openstreetmap/include/functions_map.php on line 96

Running this environment : 
(https://fr.piwigo.org/) 14.2.0 
Linux
PHP: 8.1.27
MySQL: 10.6.16-MariaDB-0ubuntu0.22.04.1 [2024-02-16 09:53:17]

**Comment by plegall** [comment]
We have made some changes yesterday about this matter in https://github.com/Piwigo/piwigo-openstreetmap/commit/cea3267fe677e4bd579ff2dd037473fca88301ca

I don't understand why you don't have this setting :-/

**Comment by DaleBCooper** [comment]
I stumbled across the same error. The fix was simple:
Go to the settings of piwigo-openstreetmap and click "Save settings".
After this the the error has gone and the plugin works again.

**Comment by pstimpel** [comment]
@plegall 

If it helps, I had the very same issue when updating piwigo to the latest version today. It was solved the same way, by saving the settings of the OSM plugin again. This resulted onetime in another php warning on the admin page, but this I was not able to reproduce. I installed the OSM years ago, but never used that settings page again after the first installation. So maybe there was an update in the past which introduced the setting, but did not take care of any migration? Sure, only a wild guess.
